outreach/write.go

```diff
@@ -0,0 +1,72 @@
+package outreach
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/url"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+type WriteResponse struct {
+	Data struct {
+		Attributes    map[string]any `json:"attributes"`
+		Relationships map[string]any `json:"relationships"`
+		Links         map[string]any `json:"links"`
+		ID            int            `json:"id"`
+		Type          string         `json:"type"`
+	} `json:"data"`
+}
+
+var JSONAPIContentTypeHeader = common.Header{ //nolint:gochecknoglobals
+	Key:   "Content-Type",
+	Value: "application/vnd.api+json",
+}
+
+type writeMethod func(context.Context, string, any, ...common.Header) (*common.JSONHTTPResponse, error)
+
+func (c *Connector) Write(ctx context.Context, config common.WriteParams) (*common.WriteResult, error) {
+	var write writeMethod
```

Comment:
this doesn't seem to be used, and once you remove it, you can also remove `writeMethod`

Reply:
write is used in creating the request method dynamically wheather it is Post or Patch. In Ln 42 or 44.

Reply:
oops I missed that.